python/mxnet/symbol.py

```diff
@@ -1025,6 +1025,36 @@ def grad(self, wrt):
         return Symbol(handle)
     # pylint: enable= no-member
 
+    def eval(self, args=None, ctx=cpu()):
+        if args == None:
```

Comment:
infershape also has this style. bind use a dict because it also need to have a dict for gradients